src/lib/plugins/feltcoop/todo/Todo.svelte

```diff
@@ -87,18 +109,28 @@
 <div class="todo">
 	<div class="entities">
 		<!-- TODO handle failures here-->
-		{#if entities && $queryStatus === 'success'}
-			<TodoItems {actor} {entities} {space} {selectedList} {selectList} />
+		{#if listsCollection && $listsCollection && querySuccess && lists}
+			<TodoItems
+				{actor}
+				parentList={listsCollection}
+				entities={lists}
+				{space}
+				{selectedList}
+				{selectList}
+			/>
 			<button
 				on:click={() =>
+					$listsCollection &&
 					actions.OpenDialog(
 						toDialogData(CreateEntityForm, {
 							done: () => actions.CloseDialog(),
-							entityName: 'todo',
+							entityName: 'todo list',
 							actor,
 							hub,
 							space,
+							type: 'OrderedCollection',
 							fields: {content: true},
```

Comment:
is the default, can also be removed from `AddListButton`

```suggestion
							fields: {content: true},
```